src/v/security/acl_store.cc

```diff
@@ -114,21 +116,79 @@ acl_store::find(resource_type resource, const ss::sstring& name) const {
     return acl_matches(wildcards, literals, std::move(prefixes));
 }
 
-void acl_store::remove_bindings(
-  const std::vector<acl_binding_filter>& filters) {
+std::vector<std::vector<acl_binding>> acl_store::remove_bindings(
```

Comment:
can we then copy the the unit tests too ?

Reply:
https://github.com/vectorizedio/redpanda/blob/dev/src/v/security/tests/authorizer_test.cc#L403